lib/color_spinor_field.cpp

```diff
@@ -186,7 +169,7 @@ namespace quda
       components.reserve(composite_descr.dim);
       for (int cid = 0; cid < composite_descr.dim; cid++) {
         param.component_id = cid;
-        param.v = static_cast<void *>(static_cast<char *>(v) + cid * bytes / composite_descr.dim);
+        param.v = static_cast<void *>(static_cast<char *>(v.data()) + cid * bytes / composite_descr.dim);
```

Comment:
`v.data<char*>()`?

Reply:
No applicable here.  There is no template option for `quda_ptr::data()` (unlike for the higher-level object constructs `data` member function).